openvoxserver-ca.gemspec

```
@@ -4,14 +4,14 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require "puppetserver/ca/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = "puppetserver-ca"
+  spec.name          = "openvoxserver-ca"
   spec.version       = Puppetserver::Ca::VERSION
-  spec.authors       = ["Puppet, Inc."]
-  spec.email         = ["release@puppet.com"]
+  spec.authors       = ["OpenVox Project"]
+  spec.email         = ["openvox@voxpupuli.org"]
   spec.license       = "Apache-2.0"
 
-  spec.summary       = %q{A simple CLI tool for interacting with Puppet Server's Certificate Authority}
-  spec.homepage      = "https://github.com/puppetlabs/puppetserver-ca-cli/"
+  spec.summary       = %q{A simple CLI tool for interacting with OpenVox Server's Certificate Authority}
+  spec.homepage      = "https://github.com/OpenVoxProject/openvoxserver-ca-cli/"
```

My proposal is to remove the `-cli` suffix for consistency:

```suggestion
  spec.homepage      = "https://github.com/OpenVoxProject/openvoxserver-ca/"
```

---

Yes, would be better if the repo and gem names are consistent.  My guess is someone added -cli to the repo name to make it clear it doesn’t actually contain the ca implementation, just a ui.